/**:
  ros__parameters:
    waypoint_file_type: "apt"
    waypoint_file_path: "/home/maciej/dev/dump/models/APT/tetraedre_3faces.apt"
    shift: [1.0, -0.7, 0.2]
    scale: 0.001
    velcocity_scaling: 0.2
    acceleration_scaling: 1.0
    plot_waypoints: true
    plot_frames: false